trezorlib/client.py

```diff
@@ -1102,6 +1102,10 @@ def self_test(self):
     def stellar_get_public_key(self, address_n):
         return self.call(proto.StellarGetPublicKey(address_n=address_n))
 
+    @expect(proto.StellarAddress)
```

Comment:
please add `@field('address')` above this line, and modify the tests accordingly.

(the equivalent should be done for stellar_get_public_key - if you have the time, please add that too :) )